src/components/common/Balance.js

```diff
@@ -8,24 +8,35 @@ const CustomDiv = styled(List)`
     display: inline;
 `
 
+const FRAC_DIGITS = 5
+
 const Balance = ({ amount }) => {
     if (!amount) {
         throw new Error("amount property should not be null")
     }
     let amountShow = formatNEAR(amount)
+
     return (
-        <CustomDiv>
+        <CustomDiv title={['0','0.0001'].includes(utils.format.formatNearAmount(amount, (FRAC_DIGITS || 1) - 1)) ? `${formatWithCommas(amount)} yoctoⓃ` : ``}>
             {amountShow} Ⓝ
         </CustomDiv>
     )
 }
 
 export const formatNEAR = (amount) => {
-    let ret =  utils.format.formatNearAmount(amount, 5);
+    let ret =  utils.format.formatNearAmount(amount, FRAC_DIGITS);
     if (ret === '0') {
-        return "<0.00001";
+        return `<${!FRAC_DIGITS ? `0` : `0.${'0'.repeat((FRAC_DIGITS || 1) - 1)}1`}`
```

Comment:
Here again, we know that `FRAC_DIGITS` always exists, right? (I don't think the returned string makes sense anyhow, in the case that `FRAC_DIGITS` doesn't exist here – I think it would return `'<01'`?)

Reply:
In case that `FRAC_DIGITS` is `0` it will return correct `<0`.